src/components/plot-list/index.tsx

```diff
@@ -21,6 +23,8 @@ export interface PlotListProps extends ActionHandler<ShelfAction|ResultAction> {
   bookmark: Bookmark;
 
   limit?: number;
+
+  isLoading: boolean;
```

Comment:
PlotList takes almost everything in Result as parameters (except query): `plots`, `limit`, `isLoading`.
Maybe we should just take `result` as the parameter instead?